src/main/java/com/stripe/param/CustomerCreateParams.java

```diff
@@ -0,0 +1,591 @@
+// Generated by com.stripe.generator.entity.SdkBuilder
+
+package com.stripe.param;
+
+import com.google.gson.annotations.SerializedName;
+import com.stripe.net.ApiRequestParams;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import lombok.Getter;
+
+@Getter
+public class CustomerCreateParams extends ApiRequestParams {
+  /**
+   * An integer amount in %s that represents the account balance for your customer. Account balances
+   * only affect invoices. A negative amount represents a credit that decreases the amount due on an
+   * invoice; a positive amount increases the amount due on an invoice.
+   */
+  @SerializedName("account_balance")
+  Long accountBalance;
+
+  @SerializedName("coupon")
+  String coupon;
+
+  /**
+   * An arbitrary string that you can attach to a customer object. It is displayed alongside the
+   * customer in the dashboard.
+   */
+  @SerializedName("description")
+  String description;
+
+  /**
+   * Customer's email address. It's displayed alongside the customer in your dashboard and can be
+   * useful for searching and tracking. This may be up to *512 characters*.
+   */
+  @SerializedName("email")
+  String email;
+
+  /** Specifies which fields in the response should be expanded. */
+  @SerializedName("expand")
+  List<String> expand;
+
+  /**
+   * The prefix for the customer used to generate unique invoice numbers. Must be 3–12 uppercase
+   * letters or numbers.
+   */
+  @SerializedName("invoice_prefix")
+  String invoicePrefix;
+
+  /** Default invoice settings for this customer. */
+  @SerializedName("invoice_settings")
+  InvoiceSettings invoiceSettings;
+
+  /**
+   * A set of key-value pairs that you can attach to a customer object. It can be useful for storing
+   * additional information about the customer in a structured format.
+   */
+  @SerializedName("metadata")
+  Map<String, String> metadata;
+
+  @SerializedName("payment_method")
```

Comment:
Bringing over discussion in previous PR
> You're missing plan here. Billing will likely say they don't want to support this and would like to deprecate it but it's actively used. Technically you can add it via the map so it's not a deal breaker but I wanted to flag, same for quantity and tax_percent and other fields. See https://github.com/stripe/stripe-dotnet/blob/master/src/Stripe.net/Services/Customers/CustomerCreateOptions.cs#L31 
>
> I'm flagging this because this will be really common. An eng team will want to undocument something but removing it from stripe-java will be a major version. And we tend to not do that in go and .net. Doing this is a "waste" of major versions and we tend to deprecate what's actively removed on an API version instead.

Reply:
Per PAYFLOWS-1293, it is supported on customer creation